Dockerfile

```
@@ -0,0 +1,25 @@
+FROM ruby:2.7-alpine
```

We can use an arg for the Ruby version here:
```suggestion
ARG RUBY_VERSION=${RUBY_VERSION:-3.2}
FROM ruby:${RUBY_VERSION}-alpine
```